map/map.js

```diff
@@ -0,0 +1,20 @@
+import { quests } from '../data.js';
+import { getUser } from '../local-storage-utils.js';
+
+const section = document.querySelector('section');
+
+const user = getUser();
+
+if (user.completed['castle'] && user.completed['fountain'] && user.completed['cave']) {
```

Comment:
This works, you can create a function to check these conditions for you if you wanted.